src/pages/ReportParticipantsPage.js

```diff
@@ -91,7 +91,7 @@ const ReportParticipantsPage = (props) => {
                     styles.containerWithSpaceBetween,
                 ]}
             >
-                {participants.length
+                {Boolean(participants.length)
```

Comment:
NAB, I usually prefer to compare length like `participants.length > 0` since that is a more explicit "boolean" version of checking the length of a string - but this works too.

Reply:
👍   I'll keep that in mind for next time.